projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -1274,7 +1274,18 @@
 
 				var item = domUtil.closest( target, itemSelector );
 				var index = Array.prototype.indexOf.call( gallery.querySelectorAll( itemSelector ), item );
-
+				if ( ! window.Swiper ) {
+					var head = document.getElementsByTagName( 'head' )[ 0 ];
+					var jsScript = document.createElement( 'script' );
+					jsScript.id = 'jetpack-carousel-js';
+					jsScript.type = 'text/javascript';
+					jsScript.src = '/wp-content/plugins/jetpack/modules/carousel/swiper-bundle.js';
```

Comment:
need to work out how to generate this path properly based on env

Reply:
You could always pass in the path to it from the PHP side? We already add the login URL to `login_url` in `$localize_strings` so there's a precedent there 🙂

Reply:
managed to do it with `wp_localize_script`